change/@fluentui-web-components-7b17a2b2-f134-4934-8331-709bea980b6c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=32116&commitId=0c61d1fe4a4390c84fe185024557b6beff7f30c0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|4| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|2824| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2232| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>